Split "Recommended Software for Computer Science" section from First Year Guide (#187)
UWindsorCSS/wiki: resources/guides/recommended-software.md

int main(void) {
    printf("Chip says hello!");
}
```

Find the file in your shell. For example, if you put the file on your Desktop, use the following command to go to there.

```sh
cd ~/Desktop
```

Compile the program into a **binary** (runnable file, executable, etc.).

-   `-Wall` tells the compiler to output all warnings.
-   `-o hello.out` tells the compiler to save the created binary with the name "hello.out".

```sh
gcc -Wall hello.c -o hello.out
```

Run the binary you just created.

```sh
./hello.out
```

The output should be

```
Chip says hello!
```

## Remote Connection: SSH and NoMachine

### Description

As a computer science student, you have access to the School of Computer Science's servers, which provide software in a Linux environment that you are free to use. On campus, they can be accessed using School of Computer Science computer labs. Remotely, they can be accessed in several ways.

-   SSH (recommended): Remotely use a shell on the server.
-   NoMachine: Remote desktop environment.

:::important

-   You must [authenticate your UWin account](https://auth.cs.uwindsor.ca/) with the School of Computer Science before you can log in remotely.

> -   To log in remotely from off-campus, you must be on the VPN first. Please refer to our guide on how to set up the VPN [here](https://uwindsorcss.github.io/wiki/resources/guides/vpn).

Reviewer: ```suggestion
-   To log in remotely from off-campus, you must be on the VPN first. Refer to the [VPN Setup Guide](./vpn).
```
